### PR TITLE
Development

### PR DESCRIPTION
**Pull request by esteban056**
Changes:
- version of mongodb
- installation key and repo to debian 8
- install commands
Adds:
- task to start service
- task to unlock mongod file

### CODE REVIEW

File: tasks/configure.yml [hunk 1]
```diff
@@ -22,6 +22,9 @@
   notify: ["mongodb restart"]
   when: mongodb_transparent_hugepage_disable
 
+- name: mongodb start
+  service: name=mongod state=started daemon-reload=true
```
Comment (AAlvz) [inline tasks/configure.yml]: rewrite following the "hash" syntax:

service: 
  name: mongod
  state: started
  daemon-reload: true

File: tasks/install.deb.yml [hunk 2]
```diff
@@ -7,13 +7,25 @@
     mongodb_repository_data: "{{mongodb_repository_config[mongodb_major_minor_version]}}"
 
 - name: Add APT key
-  apt_key: keyserver=keyserver.ubuntu.com id="{{mongodb_repository_data.key}}"
+  apt_key: 
+    keyserver: keyserver.ubuntu.com
```
Comment (AAlvz) [inline tasks/install.deb.yml]: can we parametrize this one?

**Review by AAlvz — APPROVED** [review]
LGTM

**Review by cracos — APPROVED** [review]
LGTM

**Pull request merged**